Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelResult.kt

```diff
@@ -49,8 +49,6 @@ interface RiskLevelResult {
             aggregatedRiskResult?.mostRecentDateWithLowRisk
         }
 
-    val traceLocationCheckInRiskStates: List<TraceLocationCheckInRisk>
```

Comment:
Check-In high risk can also cause a red card on the home-screen, as well as influence other parts of the app. All of these consume, the RiskLevelResult, don't we need something like that here?

Reply:
Yes, will come with a new PR